### ISSUE TITLE
CRUD Reviews

### ISSUE THREAD

**Issue by nvisal1** [Cyber4All/clark-admin#30]
Allow editors/admins to crud reviews on a learning object. The admin routes in rating service are already implemented

**Comment by DonSeannelly** [comment]
@nvisal1 can you elaborate on the specifics of each piece of functionality here? I can see allowing editors/admins to remove a review if it is inappropriate, but I'm not sure on how the CRU operations are coming to play here.

**Comment by nvisal1** [comment]
> @nvisal1 can you elaborate on the specifics of each piece of functionality here? I can see allowing editors/admins to remove a review if it is inappropriate, but I'm not sure on how the CRU operations are coming to play here.

You're correct. That was a typo. Admins/ editors should only be able to read and delete from the admin dashboard
